src/ekat/simd/ekat_simd_utils.hpp

```diff
@@ -0,0 +1,198 @@
+#ifndef EKAT_SIMD_UTILS_HPP
```

Comment:
Name and location of the file are up for debate. I think I may move the generic interface into a generic file, and add the Pack specialization to ekat_pack_utils.hpp maybe?